server/sonar-server/src/test/java/org/sonar/server/issue/index/IssueIndexTest.java

```diff
@@ -392,6 +393,7 @@ public void filter_by_resolved() {
     assertThatSearchReturnsOnly(IssueQuery.builder().resolved(null), "I1", "I2", "I3");
   }
 
+  @Ignore
   @Test
   public void filter_by_rules() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Fix or remove this skipped unit test [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1607)